aea/protocols/dialogue/base.py

```diff
@@ -217,6 +212,9 @@ class _DialogueMeta(type):
 
     def __new__(cls, name: str, bases: Tuple[Type], dct: Dict) -> "_DialogueMeta":
         """Construct a new type."""
+        # apply empty slots if no slots defined.
+        # temporary disable dct["__slots__"] = dct.get("__slots__", tuple())
```

Comment:
Not sure I understand?

Reply:
to pass some tests with dialogues missing slots

Reply:
so will remove after other fixes done